lib/GitCommands.js

```diff
@@ -54,26 +52,23 @@ class GitCommands {
 
   /**
      * Create a git tag with the given version number and optional message.
-     * @param{Object<{version, message}>} Version: Used for the tag name, message: The message to describe the tag.
+     * @param{Object<{version, message}>} Version: Used for the tag name,
+     * message: The message to describe the tag.
      * @returns Git output or error when the command failed.
      */
   async createTag({ version, message }) {
     if (typeof version === 'string' && typeof message === 'string') {
-      try {
-        if (!version || version === '') {
-          throw new Error('✖ A version is required to create a tag.');
-        }
-        if (await this.hasUncommittedChanges()) {
-          throw new Error('✖ Please commit your changes to Git first.');
-        }
-
-        const result = await this._executeGitCommand(`tag -a "v${version}" -m "${message}"`);
-
-        if (result.hasOwnProperty('stderr') && result.stderr.includes('already')) {
-          throw new Error('✖ This Git tag already exists!');
-        }
-      } catch (error) {
-        throw error;
+      if (!version || version === '') {
+        throw new Error('✖ A version is required to create a tag.');
+      }
+      if (await this.hasUncommittedChanges()) {
+        throw new Error('✖ Please commit your changes to Git first.');
+      }
+
+      const result = await this._executeGitCommand(`tag -a "v${version}" -m "${message}"`);
+
+      if (result.stderr && result.stderr.includes('already')) {
```

Comment:
```suggestion
      if (typeof result.stderr === 'string' && result.stderr.includes('already')) {
```

This is fine as well since it is always a string, just think this check gives more info to the reader.

Reply:
see https://github.com/athombv/node-homey/pull/203